<!-- MHonArc v2.6.16 -->
<!--X-Subject: Re: Monte Carlo problem with model&#45;free.py and relax1.3.3 -->
<!--X-From-R13: "Sqjneq q'Ohiretar" <rqjneq.qnhiretarNtznvy.pbz> -->
<!--X-Date: Tue, 28 Apr 2009 10:10:23 +0200 -->
<!--X-Message-Id: 7f080ed10904280109l35c27abaq79191da961060287@mail.gmail.com -->
<!--X-Content-Type: text/plain -->
<!--X-Reference: 93CF5D3F6D8F4012BD5716BEC76DD603@Gandalf -->
<!--X-Head-End-->
<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN"
   "http://www.w3.org/TR/html4/loose.dtd">
<html>
<head>
<title>Re: Monte Carlo problem with model-free.py and relax1.3.3 -- April 28, 2009 - 10:10</title>
<link rel="stylesheet" type="text/css" href="/mail.gna.org/archives-color-gna.css"> 
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
</head>
<body>
<!--X-Body-Begin-->
<!--X-User-Header-->
<!--X-User-Header-End-->
<!--X-TopPNI-->
<h2><img src="/mail.gna.org/images/mail.orig.png" width="48" height="48"
alt="mail" class="pageicon" />Re: Monte Carlo problem with model-free.py and relax1.3.3</h2>
<br />
<div class="topmenu">
<a href="../" class="tabs">Others Months</a> | <a href="index.html#00001" class="tabs">Index by Date</a> | <a href="threads.html#00001" class="tabs">Thread Index</a><br />
<span class="smaller">&gt;&gt;&nbsp;&nbsp;
[<a href="msg00000.html">Date Prev</a>] [Date Next] [<a href="msg00000.html">Thread Prev</a>] [Thread Next]
</div>

<!--X-TopPNI-End-->
<!--X-MsgBody-->
<!--X-Subject-Header-Begin-->
<h3><a name="header" href="#header">Header</a></h3>
<!--X-Subject-Header-End-->
<!--X-Head-of-Message-->
<ul class="headdata">
<li class="menuitem">
<em>To</em>: Martinek Tamas &lt;martinek@xxxxxxxxxxxxxxxxx&gt;</li>
<li class="menuitem">
<em>Date</em>: Tue, 28 Apr 2009 10:09:43 +0200</li>
<li class="menuitem">
<em>Cc</em>: relax-users@xxxxxxx</li>
<li class="menuitem">
<em>Message-id</em>: &lt;<a href="msg00001.html">7f080ed10904280109l35c27abaq79191da961060287@mail.gmail.com</a>&gt;</li>
<li class="menuitem">
<em>References</em>: &lt;<a href="msg00000.html">93CF5D3F6D8F4012BD5716BEC76DD603@Gandalf</a>&gt;</li>
</ul>
<!--X-Head-of-Message-End-->
<!--X-Head-Body-Sep-Begin-->
</div><!-- end headdata -->
<br />
<h3><a name="content" href="#content">Content</a></h3>
<div class="postedby">Posted by <strong>Edward d'Auvergne</strong> on April 28, 2009 - 10:10:</div>
<div class="msgdata">
<!--X-Head-Body-Sep-End-->
<!--X-Body-of-Message-->
<pre style="margin: 0em;">Hi,

Welcome to the relax mailing lists.  This is quite a peculiar error, I
haven't seen that one before.  Would you be able to submit a bug
report for this problem?  This can be done at
<a  rel="nofollow" href="https://web.archive.org/web/https://gna.org/bugs/?group=relax">https://gna.org/bugs/?group=relax</a>.  Thanks.  Also, if you create a
tiny data set which reproduces this error and attach all necessary
files to the report, this will allow me to very quickly find and fix
the bug.  I will then be able to add this to the test suite, helping
debugging and preventing the bug from returning.  If the data is
sensitive, then you could randomise the numbers a bit or change the
spin names, just as long as the bug is reproduced.

Cheers,

Edward


On Tue, Apr 28, 2009 at 9:55 AM, Martinek Tamas
&lt;martinek@xxxxxxxxxxxxxxxxx&gt; wrote:
</pre><blockquote class="blockquote"><pre style="margin: 0em;">

Hi,

I recently installed relax-1.3.3 from source on a RH9. The sample script
model-free.py runs nicely without monte carlo error estimaton. When the
monte carlo step is included, I get the following error:

Traceback (most recent call last):
 File &quot;/usr/local/bin/relax&quot;, line 422, in ?
   Relax()
 File &quot;/usr/local/bin/relax&quot;, line 131, in __init__
   self.interpreter.run(self.script_file)
 File &quot;/usr/local/relax/prompt/interpreter.py&quot;, line 270, in run
   return run_script(intro=self.__intro_string, local=self.local,
script_file=script_file, quit=self.__quit_flag,
show_script=self.__show_script, raise_relax_error=self.__raise_relax_error)
 File &quot;/usr/local/relax/prompt/interpreter.py&quot;, line 531, in run_script
   return console.interact(intro, local, script_file, quit,
show_script=show_script, raise_relax_error=raise_relax_error)
 File &quot;/usr/local/relax/prompt/interpreter.py&quot;, line 427, in
interact_script
   execfile(script_file, local)
 File &quot;model-free.py&quot;, line 78, in ?
   monte_carlo.initial_values()
 File &quot;/usr/local/relax/prompt/monte_carlo.py&quot;, line 165, in initial_values
   monte_carlo.initial_values()
 File &quot;/usr/local/relax/generic_fns/monte_carlo.py&quot;, line 269, in
initial_values
   init_sim_values()
 File &quot;/usr/local/relax/specific_fns/model_free/main.py&quot;, line 2938, in
sim_init_values
   object = getattr(cdp, object_name)
AttributeError: 'PipeContainer' object has no attribute 'chi2'

The chi2 is properly calculated during the first minimization before MC, and
dumped in the results file. I get this error message with the test data
(noe.500.out, r1.500.out, etc.) too.

Thanks!

Tamas




=====================================
Tamas Martinek, PhD
associate professor
Institute of Pharmaceutical Chemistry
University of Szeged
6720 Szeged Eotvos u. 6, Hungary
phone: +36 62 545768
fax:   +36 62 545705
=====================================


_______________________________________________
relax (<a  rel="nofollow" href="http://nmr-relax.com">http://nmr-relax.com</a>)

This is the relax-users mailing list
relax-users@xxxxxxx

To unsubscribe from this list, get a password
reminder, or change your subscription options,
visit the list information page at
<a  rel="nofollow" href="/mail.gna.org/listinfo/relax-users">https://mail.gna.org/listinfo/relax-users</a>

</pre></blockquote><pre style="margin: 0em;">


</pre>
<!--X-Body-of-Message-End-->
<!--X-MsgBody-End-->
<!--X-Follow-Ups-->
</div><!-- end msgdata -->
<br />
<h3><a name="related" href="#related">Related Messages</a></h3>
<div class="relateddata">
<!--X-Follow-Ups-End-->
<!--X-References-->
<ul><li><strong>References</strong>:
<ul>
<li><strong><a name="00000" href="msg00000.html">Monte Carlo problem with model-free.py and relax1.3.3</a></strong>
<ul><li><em>From:</em> Martinek Tamas</li></ul></li>
</ul></li></ul>
<!--X-References-End-->
<!--X-BotPNI-->
</div><!-- end relateddata -->
<!-- NoBotLinksApartFromRelatedMessages -->

<!--X-BotPNI-End-->
<!--X-User-Footer-->
<!--X-User-Footer-End-->
<div class="footer"></div><br />
<div class="right">Powered by <a href="http://www.mhonarc.org">MHonArc</a>, Updated Tue Apr 28 10:21:24 2009</div>  
</body>
</html>
